stacks-blockchain.md

```diff
@@ -33,17 +33,20 @@ rpc_bind = "0.0.0.0:20443"
 p2p_bind = "0.0.0.0:20444"
 bootstrap_node = "02196f005965cebe6ddc3901b7b1cc1aa7a88f305bb8c5893456b8f9a605923893@seed.mainnet.hiro.so:20444"
 seed = "<npx privateKey from wallet.md>"
-# local_peer_seed = "" ## this value isn't required and is only used in peer networking
+local_peer_seed =  "<npx privateKey from wallet.md>"
```

Comment:
revert. that's not what this key does, and is only required for a seed node. miners/follwers don't need this